docs/blog/2018-10-26-introducing-gatsby-themes/index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Introducing Gatsby Themes
+date: 2018-10-26
+author: Chris Biscardi
+excerpt: "Scaling the production of Gatsby sites"
+tags: ["themes", "architecture"]
+---
+
+Gatsby is a powerful platform for building marketing sites, blogs,
+ecommerce frontends, and more. We can source data from static files
```

Comment:
Just noticing a couple more "we"'s that confused me. Another chance for "you", if you ask me ;)

Reply:
heh, habits die hard.

Reply:
@shannonbux handled.